include/aspect/utilities.h

```diff
@@ -551,10 +551,35 @@ namespace aspect
     /**
      * Checks whether a file named @p filename exists and is readable.
      *
+     * Warning: This function performs file access on all MPI ranks that
+     * call it. This can lead to overloading the file system if the
+     * function is called on many MPI ranks. Only use this function if
+     * you are sure that the file access is performed on all MPI ranks
+     * and on a fast file system (e.g. a local drive).
+     *
+     * @return True if the file exists and is readable on all MPI ranks
+     * that call this function.
      * @param filename File to check existence
      */
     bool fexists(const std::string &filename);
 
+    /**
+     * Checks whether a file named @p filename exists and is readable
+     * on MPI rank 0. Afterwards, the result is broadcasted to all MPI
+     * ranks.
+     *
+     * Note that in contrast to the other fexists function, this function
+     * uses MPI to check the existence of the file only on MPI rank 0.
+     * This is useful to avoid overloading the file system and is sufficient
```

Comment:
```suggestion
     * uses MPI to check the existence of the file only on MPI rank 0 and
     * broadcasts the results to all ranks.
     * This is useful to avoid overloading the file system and is sufficient
```